compiler/rustc_typeck/src/check/pat.rs

```diff
@@ -1468,6 +1469,18 @@ impl<'a, 'tcx> FnCtxt<'a, 'tcx> {
                         // We don't want to throw `E0027` in case we have thrown `E0026` for them.
                         unmentioned_fields.retain(|&(_, x)| x.name != suggested_name);
                     }
+                } else if inexistent_fields.len() == 1 {
+                    let unmentioned_field = unmentioned_fields[0].1.name;
+                    err.span_suggestion(
```

Comment:
One small change so that w e don't mention the field name twice in a row:
```suggestion
                    err.span_suggestion_short(
```

After that, r=me.

Reply:
Applied :+1: I don't have the bors permission so please do it instead of me.